.github/workflows/build.yml

```diff
@@ -92,12 +103,21 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
+      - name: Login to Quay
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_TOKEN }}
+
       - name: Build & Push Image
-        env:
-          IMAGE_ORG: ${{ secrets.IMAGE_ORG}}
-        run: |
-          make docker.buildx.ndm
-          make buildx.push.ndm
+        uses: docker/build-push-action@v2
+        with:
+          file: ./build/ndm-daemonset/Dockerfile
+          push: true
+          platforms: linux/amd64, linux/arm64, linux/arm/v7, linux/ppc64le
+          tags: |
+            ${{ steps.docker_meta.outputs.tags }}
```

Comment:
Should we pass the build-args as well? I see there are a few build-args in the Dockerfile

Reply:
Done.